client/errors.go

```diff
@@ -17,6 +17,8 @@ var (
 	ErrMissingObject = errors.New("missing object")
 	// ErrMissingAccount is returned when account/owner is not provided.
 	ErrMissingAccount = errors.New("missing account")
+	// ErrMissingSigner is returned when signer is not provided.
+	ErrMissingSigner = errors.New("missing signer")
```

Comment:
Well, this implies that other signer users should check for it too, which is an additional patch.

Reply:
do you mean another PR or another commit for that? we can fix panic in that PR and add `ErrMissingSigner` in another (or just create an issue for that)

Reply:
ok, added in that PR